README.md

```diff
@@ -118,45 +160,99 @@ This library has a `peerDependencies` listing for `react-dom`.
+  setupTestFrameworkScriptFile: require.resolve('./jest.setup.js'),
+  // ... other options ...
+}
+```
+
+```javascript
+// jest.setup.js
+
+// add some helpful assertions
 import 'jest-dom/extend-expect'
-import axiosMock from 'axios' // the mock lives in a __mocks__ directory
-import Fetch from '../fetch' // see the tests for a full implementation
 
-// automatically unmount and cleanup DOM after the test is finished.
-afterEach(cleanup)
+// this is basically: afterEach(cleanup)
+import 'react-testing-library/cleanup-after-each'
+```
 
-test('Fetch makes an API call and displays the greeting when load-greeting is clicked', async () => {
-  // Arrange
-  axiosMock.get.mockResolvedValueOnce({data: {greeting: 'hello there'}})
-  const url = '/greeting'
-  const {getByText, getByTestId, container} = render(<Fetch url={url} />)
+### Custom Render
 
-  // Act
-  fireEvent.click(getByText('Load Greeting'))
+It's often useful to define a custom render method that includes things like
+global context providers, data stores, etc. To make this available globally, one
+approach is to define a utility file that re-exports everything from
+`react-testing-library`. You can replace react-testing-library with this file in
+all your imports.
 
-  // let's wait for our mocked `get` request promise to resolve
-  // wait will wait until the callback doesn't throw an error
-  const greetingTextNode = await waitForElement(() =>
-    getByTestId('greeting-text'),
+```diff
+// my-component.test.js
+- import { render, fireEvent } from 'react-testing-library';
++ import { render, fireEvent } from '../test-utils';
+```
+
+```js
+// test-utils.js
+import {render} from 'react-testing-library'
+import {ThemeProvider} from 'my-ui-lib'
+import {TranslationProvider} from 'my-i18n-lib'
+import defaultStrings from 'i18n/en-x-default'
+
+const customRender = (node, ...options) => {
+  return render(
+    <ThemeProvider theme="light">
+      <TranslationProvider messages={defaultStrings}>
+        {node}
+      </TranslationProvider>
+    </ThemeProvider>,
+    ...options,
   )
+}
 
-  // Assert
-  expect(axiosMock.get).toHaveBeenCalledTimes(1)
-  expect(axiosMock.get).toHaveBeenCalledWith(url)
-  expect(getByTestId('greeting-text')).toHaveTextContent('hello there')
-  expect(getByTestId('ok-button')).toHaveAttribute('disabled')
-  // snapshots work great with regular DOM nodes!
-  expect(container.firstChild).toMatchSnapshot()
-})
+// re-export everything
+export * from 'react-testing-library'
+
+// override render method
+export {customRender as render}
 ```
 
+To make this file accessible without using relative imports, add the folder
+containing the file to the Jest `moduleDirectories` option.
+
+```diff
+// my-component.test.js
+- import { render, fireEvent } from '../test-utils';
++ import { render, fireEvent } from 'test-utils';
+```
+
+```diff
+// jest.config.js
+module.exports = {
+  moduleDirectories: [
+    'node_modules',
++    __dirname, // the project root
```

Comment:
Let's make the comment say: "the directory where your `test-utils.js` file resides."

I'd prefer to recommend that file exists in a directory called `test` rather than at the root of the project.